server/domain/status/Status.js

```diff
@@ -170,6 +172,19 @@ class Status {
 
         return new Date() - this.data.time < expireTime;
     }
+
+    static formatUserImage(data) {
+        if (data.userImage.indexOf('gravatar.com') === -1) {
+            return;
+        }
+
+        let questionMarkLocation = data.userImage.indexOf('?');
+        if (questionMarkLocation !== -1) {
+            data.userImage = data.userImage.substring(0, questionMarkLocation);
+        }
+
+        data.userImage += '?size=300&default=mp';
```

Comment:
Please actually return the mutated object, this "magic" in memory adjustments are hard to grasp for our weak brains,

Reply:
Done!